pages/phases/job-announcement/index.html

```diff
@@ -73,8 +73,8 @@ <h2>Final checklist before JOA Posting</h2>
 <li>Do you have a link to the questionnaire in your JOA?</li>
 <li>Did you consider including language telling applicants the announcement will remain open for either 240 days or 365 days to maximize selections off the certificate throughout the year?</li>
 <li>Are the SMEs and HMs ready to do recruitment when you send them the link?</li>
-<li>Is your HR speicialist in office and ready to process applicants any day the application may close?</li>
-<li>Is your announcement clear about the locations of each posting and if you can hire people remotely? (candidates may assume positions are remote-friendly)</li>
+<li>Have you gamed out availability and days off for your coordinator and HR specialist so that the assessment process can proceed smoothly?
```

Comment:
```suggestion
<li>Have you fully analyzed potential availability and days off for your coordinator and HR specialist so that the entire hiring process can proceed without unnecessary delays?
```